smrf/data/hrrr_grib.py

```diff
@@ -111,7 +115,7 @@ def load_timestep_thread(self, date_times, data_queue):
         for date_time in date_times[1:]:
             self.load_timestep(date_time)
 
-            for variable in self.variables:
+            for variable in self.data_variables:
```

Comment:
This was the major issue. The `variables` were filtered to non-wind variables where the `data_queue` was expecting there to be values to get (even if they aren't used).

Reply:
Do I understand this correct that the threaded solution will revert back to loading the data even though not used?

Reply:
It's not loading wind so it keeps your PR #209. On line 78 it still filters out the wind variables so they aren't loaded. This just ensures that the `data_queue` is populated using the empty dataframes set in `InputGribHRRR.calculate_wind`

Reply:
Got it. Mainly asking to make sure there is consistent behavior independent of running threaded or not.